libraries/ms-drivers/src/adt7476a_fan_controller.c

```diff
@@ -101,3 +102,174 @@ StatusCode adt7476a_init(Adt7476aStorage *storage, Adt7476aSettings *settings) {
 
   return STATUS_CODE_OK;
 }
+
```

Comment:
so we recently changed this structure in MPXE, we're now gonna try to just use #ifdefs within the original code since that should make it cleaner and more obvious what's going on in MPXE vs not in MPXE, so rather than having the whole file in a #else / #endif, just take the MPXE specific parts and add them to the original code with a #ifdef / #endif. There should be examples in drivers on the `mpxe` branch. Sorry about this, we're still ironing out the details.

Reply:
I think you might've misunderstood what I meant by #ifdefs within the original code, check the mcp2515.c driver on `mpxe` branch for what I mean by specific parts, and check with me if you have questions